Xamarin/SSW.Consulting/SSW.Consulting/Views/LeaderBoard.xaml.cs

```diff
@@ -60,32 +63,41 @@ private void SearchButtonPress(object sender, EventArgs e)
 
         private void UnfocusSearchBar(bool debounce)
         {
+            dismiss = DateTime.Now;
+
             var shouldDismiss = debounce ? (DateTime.Now - dismiss).Duration().Milliseconds > 600 : true;
             if (searchView.IsFocused && shouldDismiss)
             {
                 searchView.Unfocus();
-                dismiss = DateTime.Now;
             }
         }
 
         private void ToggleSearchBar()
         {
-            
-                if (showBar)
+            var shouldDismiss = (DateTime.Now - show).Duration().Milliseconds > 200;
+            if (!shouldDismiss)
+            {
+                return;
+            }
+            if (showBar)
                 {
                     searchFrame.FadeTo(0, 100);
                     searchFrame.TranslateTo(0, -50, 150);
                     leaderList.Margin = new Thickness(0, 0, 0, 0);
                     showBar = false;
-                }
-                else
+                show = DateTime.Now;
+
+            }
+            else
                 {
                     searchFrame.FadeTo(1, 150);
                     searchFrame.TranslateTo(0, 0, 100);
                     leaderList.Margin = new Thickness(0, 60, 0, 0);
                     showBar = true;
-                }
-            
+                show = DateTime.Now;
+
```

Comment:
@jackkranz make sure you're cleaning up empty lines at the end of methods for better code readability

Reply:
noted